docs/index.rst

```diff
@@ -18,3 +18,4 @@ documentation for accessing the cloud data.
    accessing_data
    pangeo_catalog
    licensing_citation
+   presentations
```

Comment:
```suggestion
   presentation
```

Reply:
This won't work unless you give the document the correct name

Reply:
sure, I updated the typo seeing the preview page, but unable to check if its reflected correctly now?

Reply:
looks ok now https://raw.githack.com/pangeo-data/pangeo-cmip6-cloud/presentations-preview/index.html